mac-setup.sh

```diff
@@ -298,7 +285,7 @@ install_helpful_tools() {
 
 echo "\n"
 success "Running Khan Installation Script 1.1\n"
-warn "Warning: This is only tested on Mac OS 10.9 (Mavericks)\n"
+warn "Warning: This is only tested on Mac OS 10.9 (Mavericks)\nand Mac OS 10.10 (Yosemite)\n"
```

Comment:
:lollipop: we should really test this from start to finish again. i'll ask karina for the dev laptop